grpc/protos/README.md

```diff
@@ -0,0 +1,9 @@
+# Webserver protobuf definitions
+
+This directory contains protobuf definitions for the webserver gRPC API.
+
+To generate the Go code, run this script from the repository root:
+
+```sh
+./gen-proto.sh
```

Comment:
nit: will it be surprising that this generates the code for everything, not just the given package? What if we just have this in the top-level readme instead?

Reply:
It's a little surprising, but this is exactly the place where I'd expect instructions or a README to live (alongside the proto defs). I'll just add a note about how this script will regenerate everything.